dev/README_RELEASE_AIRFLOW.md

```diff
@@ -583,7 +621,7 @@ https://pypi.python.org/pypi/apache-airflow
 
 The documentation is available on:
 https://airflow.apache.org/
-https://airflow.apache.org/docs/${VERSION}/
+https://airflow.apache.org/docs/apache-airflow/${VERSION}/
```

Comment:
Something about having `airflow` appear in the URL twice bugs me unduly, but I don't really have an alternate suggestion (and I agree that having providers separately is right)

Oh well.

Reply:
I also have no idea for a better directory structure. Now we will be able to create a documentation link easily based on the package name in the pypi repository.

https://airflow.apache.org/docs/<PYPII-NAME>/${VERSION}/